### ISSUE TITLE
Announce new LogList quotes

### ISSUE THREAD

**Issue by ForNeVeR** [codingteam/horta-hell#308]
I think that's good idea to announce when the new quote get approved. Currently @rexim does this by hand.

**Comment by rexim** [comment]
I think if we combine #244 and https://github.com/codingteam/loglist/issues/34 we will get the desired feature.

**Comment by ForNeVeR** [comment]
Well, that's right I think. But let the issue be here so we don't forget to enable it after all preparations (i.e. implementing of #244 and codingteam/loglist#34).

**Comment by rexim** [comment]
Yeah, of course! That's correct. Getting this exact feature is a separate issue. :)
